sql/pom.xml

```diff
@@ -290,7 +290,7 @@
               <goal>exec</goal>
             </goals>
             <configuration>
-              <executable>/bin/bash</executable>
+              <executable>bash</executable>
```

Comment:
why this change ? just need to understand

Reply:
It [failed](https://github.com/rilldata/rill-developer/runs/8068489164?check_suite_focus=true) on Windows because the path to `bash` is different there

Reply:
ok got it